Datadog.Trace.Debugger.slnf

```diff
@@ -2,21 +2,18 @@
   "solution": {
     "path": "Datadog.Trace.sln",
     "projects": [
-      "shared\\src\\Datadog.Trace.ClrProfiler.Native\\Datadog.Trace.ClrProfiler.Native.vcxproj",
-      "tracer\\src\\Datadog.Tracer.Native\\Datadog.Tracer.Native.DLL.vcxproj",
-      "tracer\\src\\Datadog.Tracer.Native\\Datadog.Tracer.Native.vcxproj",
+      "tracer\\build\\_build\\_build.csproj",
+      "tracer\\src\\Datadog.InstrumentedAssemblyGenerator\\Datadog.InstrumentedAssemblyGenerator.csproj",
+      "tracer\\src\\Datadog.InstrumentedAssemblyVerification\\Datadog.InstrumentedAssemblyVerification.csproj",
       "tracer\\src\\Datadog.Trace.SourceGenerators\\Datadog.Trace.SourceGenerators.csproj",
       "tracer\\src\\Datadog.Trace.Tools.Analyzers\\Datadog.Trace.Tools.Analyzers.csproj",
       "tracer\\src\\Datadog.Trace\\Datadog.Trace.csproj",
+      "tracer\\src\\Datadog.Tracer.Native\\Datadog.Tracer.Native.DLL.vcxproj",
+      "tracer\\src\\Datadog.Tracer.Native\\Datadog.Tracer.Native.vcxproj",
       "tracer\\test\\Datadog.Trace.ClrProfiler.IntegrationTests\\Datadog.Trace.ClrProfiler.IntegrationTests.csproj",
       "tracer\\test\\Datadog.Trace.TestHelpers\\Datadog.Trace.TestHelpers.csproj",
-      "tracer\\test\\Datadog.Trace.ClrProfiler.Managed.Tests\\Datadog.Trace.ClrProfiler.Managed.Tests.csproj",
       "tracer\\test\\Datadog.Trace.Tests\\Datadog.Trace.Tests.csproj",
-      "tracer\\test\\test-applications\\debugger\\Samples.Probes\\Samples.Probes.csproj",
-      "tracer\\build\\_build\\_build.csproj",
-      "tracer\\src\\Datadog.Trace.Tools.Analyzers\\Datadog.Trace.Tools.Analyzers.csproj",
-      "tracer\\src\\Datadog.InstrumentedAssemblyGenerator\\Datadog.InstrumentedAssemblyGenerator.csproj",
-      "tracer\\src\\Datadog.InstrumentedAssemblyVerification\\Datadog.InstrumentedAssemblyVerification.csproj"
+      "tracer\\test\\test-applications\\debugger\\Samples.Probes\\Samples.Probes.csproj"
```

Comment:
You've removed a bunch of things here that you probably didn't mean to, please bring them back.

Reply:
They are still there but in correct order. Except `Datadog.Trace.ClrProfiler.Managed.Tests.csproj` that I don't think we need there.

Reply:
I think probably best to keep that one as well, as we've started adding our own unit tests to it.

Reply:
Fixed in e47f446a0c3c7a06721fdd476c2fd32799be0819